src/components/styleElements/icons/ChevronLeft.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import './Icon.css';
 
-export function ChevronLeft({ size }) {
+export function ChevronLeft({ size, className }) {
```

Comment:
I have noticed that some of the icons require very specific colors that are not either of the default text colors so I had to adjust this one to take a prop that will let me change it's color to be more specific